### PR TITLE
Add initial 1.15 Release Team members

### PR DESCRIPTION
**Pull request by justaugustus**
Signed-off-by: Stephen Augustus <saugustus@vmware.com>
ref: #545
/assign @claurence 
/hold

### CODE REVIEW

File: releases/release-1.15/release_team.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+| **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
+| ------ | ------ | ------ |
+| Lead |  Claire Laurence ([@claurence](https://github.com/claurence)) | Lachlan Evenson ([@lachie83](https://github.com/lachie83)), Niko Penteridis ([@nikopen](https://github.com/nikopen)) |
```
Comment (spiffxp) [inline releases/release-1.15/release_team.md]: @tpepper as Release Lead Shadow?

**Comment by vllry** [comment]
I'd like to shadow something, willing to backfill.

**Comment by justaugustus** [comment]
@vllry -- thanks for volunteering! I won't forget about you; we're just targeting Lead roles and some shadows at the moment. Once those are locked in, we'll start recruiting.

**Comment by spiffxp** [comment]
I see all lead slots filled, what remains to be done to get us to merged here?

**Comment by justaugustus** [comment]
@spiffxp -- @claurence will need to `/lgtm` and then we can pull the hold.

**Pull request merged**